### DISCUSSION TITLE
[tags，categories]

### DISCUSSION THREAD

**Discussion by memorieswan** [YunYouJun/hexo-theme-starry#135] [Q&A 疑难杂症] [answered]
tags和categories无法显示相关内容，但是tags 和 categories 在左侧栏 信息统计正常

![image](https://user-images.githubusercontent.com/68325149/124238828-9e4feb80-db4b-11eb-88b6-428a30e0250d.png)
![image](https://user-images.githubusercontent.com/68325149/124238894-b293e880-db4b-11eb-8274-bb7cc8b37b83.png)
点击文章中tag可以成功索引

https://memorieswan.github.io/categories/

参照链接

由 Hexo 驱动 v5.4.0|主题 - Yun v1.6.1

**Comment by YunYouJun** [comment, marked as answer]
你需要设置 categories 页面的 `type: categories`。
